spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/VndErrorResponseErrorHandler.java

```diff
@@ -60,6 +61,18 @@ public void handleError(ClientHttpResponse response) throws IOException {
 		catch (Exception e) {
 			super.handleError(response);
 		}
-		throw new DataFlowClientException(vndErrors);
+		if (vndErrors != null) {
+			throw new DataFlowClientException(vndErrors);
+		}
+		else {
+			//FIXME Basic Authentication Exceptions do not return a VndError
```

Comment:
Can we create a story for this?

Reply:
Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/2898